### PR TITLE
Fix gallery API doc URL launcher

### LINKED ISSUE #30450: Gallery app bug - "API Documentation" action broken
Steps to reproduce 
1. Launch app
2. Tap Material
3. Tap "Bottom app bar" or most/all others
4. Tap the list looking thing with tooltip of "API Documentation"
Expected result: Something in the middle of the screen
Observed result: hamburger three lines in upper left, label (e.g. Bottom Appbar) in app bar title, permanent snackbar with flutter version info at the bottom, middle of screen empty white canvas

### PR DESCRIPTION
**Pull request by HansMuller**
Fixes https://github.com/flutter/flutter/issues/30450

### CODE REVIEW

File: examples/flutter_gallery/lib/gallery/demo.dart [hunk 1]
```diff
@@ -115,6 +135,7 @@ class TabbedComponentDemoScaffold extends StatelessWidget {
                   Expanded(child: demo.demoWidget),
                 ],
               ),
+
```
Comment (xster) [inline examples/flutter_gallery/lib/gallery/demo.dart]: unintended line?

**Review by xster — COMMENTED** [review]
LGTM

File: examples/flutter_gallery/lib/gallery/demo.dart (hunk 1 above)
Comment (HansMuller) [reply examples/flutter_gallery/lib/gallery/demo.dart]: Yes, thanks

**Comment by HansMuller** [comment]
Build is actually green: https://github.com/flutter/flutter/pull/30452#issuecomment-479633322

**Pull request merged**